translations/translations.json

```diff
@@ -1227,32 +1227,20 @@
   },
   "feedback": {
     "de": {
-      "technicalTopics": "Technische Funktionen",
```

Comment:
:+1: Thanks!

Reply:
Why did you only remove the German translations?

Reply:
I thought they are the "source", when adding translations we also add only german. Is it necessary to manually remove all languages?

Reply:
I think so, yes.

Reply:
done